frontend/src/global_styles/content/_pagination.sass

```diff
@@ -28,87 +28,88 @@
 
 $pagination--font-size: 0.8125rem
 
-.pagination
+.op-pagination
   display: flex
   justify-content: space-between
   width: 100%
   min-height: 45px
 
-.pagination--pages
-  flex-grow: 2
-  flex-shrink: 2
-  margin: 10px 5px 10px 0
+  &--pages
+    flex-grow: 2
+    flex-shrink: 2
+    margin: 10px 5px 10px 0
 
-.pagination--options
-  flex-grow: 1
-  flex-shrink: 1
-  margin: 10px 0 0 5px
+  &--options
+    flex-grow: 1
+    flex-shrink: 1
+    margin: 10px 0 0 5px
 
-.pagination--items
-  list-style-type: none
-  display: flex
-  margin: 0
-  padding: 0
-  font-size: $pagination--font-size
-
-  .pagination--pages &
-    justify-content: flex-start
+  &--items
+    list-style-type: none
+    display: flex
+    margin: 0
+    padding: 0
+    font-size: $pagination--font-size
 
-  .pagination--options &
-    justify-content: flex-end
+    .op-pagination--pages &
+      justify-content: flex-start
 
-.pagination--item
-  min-width: 25px
-  margin: 0 5px 0 0
-  white-space: nowrap
+    .op-pagination--options &
+      justify-content: flex-end
 
-  &.-prev,
-  &.-next
+  &--item
     min-width: 25px
+    margin: 0 5px 0 0
+    white-space: nowrap
+
+    &_prev,
+    &_next
+      min-width: 25px
+
+    &_current
+      background: #ffffff
+      border: 1px solid #ffffff
+      padding: 3px 3px
+      font-style: italic
+      text-align: center
+
+    button, a
```

Comment:
Only select classes, otherwise these selectors will be of specificity 1 class + 1 element, instead of just one class.

something like `&--item-link` maybe.

Reply:
👍